core/templates/dev/head/components/state-editor/state-interaction-editor/state-interaction-editor.directive.ts

```diff
@@ -444,7 +444,7 @@ angular.module('oppia').directive('stateInteractionEditor', [
             $scope.customizationModalReopened = false;
 
             $scope.userBlueImgUrl = UrlInterpolationService.getStaticImageUrl(
-              '/avatar/user_blue_72px.png');
+              '/avatar/user_blue_72px.webp');
             $scope.userBlackImgUrl = UrlInterpolationService.getStaticImageUrl(
               '/avatar/user_black_72px.png');
```

Comment:
There is no file named user_black_72px.png. Probably that's why its left unchanged

Reply:
Does the variable actually get used anywhere? If not, let's remove it (but please check first).

Reply:
No this variable is only used in this file! So, I am removing it if anyone has no objections.

Reply:
No objections. Just make sure that nothing breaks -- but if that's the case, then it's actually preferable to remove, since we don't want the codebase to have dead code. Thanks!